examples/mcp4728_generalcalltest.py

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
```

Comment:
You could change this with you name or your github user if you like

Reply:
made those suggested changes